spring-integration-smb/src/main/java/org/springframework/integration/smb/session/SmbSessionFactory.java

```diff
@@ -49,7 +51,11 @@ public final SmbSession getSession() {
 	}
 
 	protected SmbSession createSession() throws IOException {
-		SmbShare smbShare = new SmbShare(this);
+		Properties props = new Properties();
+		props.setProperty("jcifs.smb.client.minVersion", this.getSmbMinVer().name());
+		props.setProperty("jcifs.smb.client.maxVersion", this.getSmbMaxVer().name());
```

Comment:
Aren't there constants for these keys ?

Reply:
No not according to the documentation for https://github.com/AgNO3/jcifs-ng which the jcifs library was forked from. Surprisingly the code between jcifs and jcifs_ng is pretty well exactly the same. I can create an enum for internal code to use if that would make it more readable.

Reply:
No, that's OK, as long as I am concerned we have only one place where we use this literals.